synapse/config/key.py

```diff
@@ -59,21 +59,31 @@ def read_config(self, config):
 
         self.expire_access_token = config.get("expire_access_token", False)
 
+        # a secret which is used to calculate HMACs for form values, to stop
+        # falsification of values
+        self.form_secret = config.get("form_secret", None)
```

Comment:
I wonder if its better to have this set close to the rest of the consent config?

Reply:
yeah, though I rather envisaged it wouldn't be limited to just consent.